### PR TITLE
OAG-96 Header, basic layout styling and upload page wireframe

### PR DESCRIPTION
**Pull request by gingerbreadpup**

### CODE REVIEW

File: app/Resources/views/base.html.twig [hunk 1]
```diff
@@ -7,16 +7,28 @@
         {% block stylesheets %}
             <link href="{{ asset('bundles/oag/stylesheets/normalize.css') }}" rel="stylesheet" />
```
Comment (ob6160) [inline app/Resources/views/base.html.twig]: This needs to be replaced with the new method of including stylesheets (through assetic) see https://github.com/NeonOpenAG/oagtoolbox/blob/9ceda088f8d31a16018cdcfcad4c0a2e35dd1914/src/OagBundle/Resources/views/Default/index.html.twig#L6 for an example.

File: app/Resources/views/base.html.twig [hunk 2]
```diff
@@ -7,16 +7,28 @@
...
             <link href="{{ asset('bundles/oag/stylesheets/main.css') }}" rel="stylesheet" />
+            <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
```
Comment (ob6160) [inline app/Resources/views/base.html.twig]: Do we still need this external dependency if the font files have been added locally?

File: src/OagBundle/Resources/public/sass/main.scss [hunk 3]
```diff
@@ -3,92 +3,201 @@
...
+  &:visited {
+    text-decoration: underline;
+  }
+  &:hover {
+    color: $oag_link;
+    text-decoration: underline;
+  }
+}
+
+// button {
```
Comment (ob6160) [inline src/OagBundle/Resources/public/sass/main.scss]: Can we remove or are you planning on keeping for something else?

File: src/OagBundle/Resources/views/Default/index.html.twig [hunk 4]
```diff
@@ -16,96 +18,134 @@
...
+
+            <h1 class="dark">Get Started</h1>
+
+            <div class="upload-form">
+                <h3>Import a new source document</h3>
+                <hr>
+                <div>The imported document will be automatically processed with CoVE and added to the IATI document list above.</div>
+                <div>Currently only XML is supported.</div>
+                <div>
+                    {{ form_start(source_upload_form) }}
```
Comment (ob6160) [inline src/OagBundle/Resources/views/Default/index.html.twig]: Is this repeated from the earlier form code on line 61? Can we use the fileUploadWidget or something similar to reduce repetition?

File: src/OagBundle/Resources/views/Default/index.html.twig [hunk 5]
```diff
@@ -123,4 +163,5 @@
             });
         });
     </script>
+    <script>
```
Comment (ob6160) [inline src/OagBundle/Resources/views/Default/index.html.twig]: Rogue opening script tag?

File: app/Resources/views/base.html.twig (hunk 2 above)
Comment (gingerbreadpup) [reply app/Resources/views/base.html.twig]: Nope

File: src/OagBundle/Resources/public/sass/main.scss (hunk 3 above)
Comment (gingerbreadpup) [reply src/OagBundle/Resources/public/sass/main.scss]: Keeping for later. :+1:

**Comment by gingerbreadpup** [comment]
So the font path seems to be correct, but the font isn't loading. Perhaps it's something to do with the Assetic thing that we're using now?

**Pull request merged**